src/Core/AutoloadInterceptor/ClassLoader.php

```diff
@@ -58,6 +58,11 @@ public function findFile($namespacedClass): false|string
 
         $filePath = Path::resolve($filePath);
 
+        foreach ($this->options->getExcludePaths() as $path) {
+            if (str_starts_with($filePath, $path)) {
```

Comment:
Here the same, `Path::resolve($path)`, so that relative paths and slight differences in paths match.